### PR TITLE
chore: get sanchonet services ready for migration

### PR DESCRIPTION
**Pull request by bernokl**
# Context

Migrating dev-sanchonet from old to new clusters

### CODE REVIEW

File: nix/cardano-services/deployments/default.nix [hunk 1]
```diff
@@ -200,9 +200,11 @@ in
         };
       };
 
-      "dev-sanchonet@us-east-1@v1" = final: {
-        name = "${final.namespace}-cardanojs-v1";
+      "dev-sanchonet@us-east-1@v1" = final: let
+        oci = inputs.cardanojs-dev-sanchonet.x86_64-linux.cardano-services.oci-images.cardano-services;
+   		in {
```
[the remark this answers is not in the document]
Comment (bernokl) [reply nix/cardano-services/deployments/default.nix]: Thanks for catching that, I am used to it being part of commit pre-hook in wallet-world

File: nix/cardano-services/deployments/default.nix [hunk 2]
```diff
@@ -219,7 +220,7 @@ in
         };
 
         values = {
-          network = "sanchonet-1";
+          network = "sanchonet";
```
Comment (michalrus) [inline nix/cardano-services/deployments/default.nix:223]: I think this is the only change?

**Review by michalrus — APPROVED** [review]
LGTM, but `treefmt` is a bit confusing, I don't know if devs run it regularly, because our run in this PR changed some TypeScript files 🤔

**Comment by iccicci** [comment]
> LGTM, but `treefmt` is a bit confusing, I don't know if devs run it regularly, because our run in this PR changed some TypeScript files 🤔

Nope, we don't use it, we _lint_ the code through `eslint`.

File: .gitignore [hunk 3]
```diff
@@ -27,6 +27,7 @@ source
 
 # direnv
 /.direnv
+/.access.local
```
Comment (gytis-ivaskevicius) [inline .gitignore:30]: everything should be in .envrc.local?

**Review by gytis-ivaskevicius — DISMISSED** [review]

File: .gitignore (hunk 3 above)
Comment (bernokl) [reply .gitignore]: Fair point, removed the entry

**Comment by michalrus** [comment]
> > LGTM, but `treefmt` is a bit confusing, I don't know if devs run it regularly, because our run in this PR changed some TypeScript files 🤔
> 
> Nope, we don't use it, we _lint_ the code through `eslint`.

@iccicci but you do use `prettier` for code formatting? This is what `treefmt` is [configured to use](https://github.com/input-output-hk/cardano-js-sdk/blob/fbd851a3be28652356777d303f2025e9a56d7159/treefmt.toml#L8-L9).

**Review by michalrus — APPROVED** [review]

File: nix/cardano-services/deployments/default.nix (hunk 2 above)
Comment (bernokl) [reply nix/cardano-services/deployments/default.nix:223]: correct

**Comment by iccicci** [comment]
> you do use `prettier` for code formatting? This is what `treefmt` is [configured to use](https://github.com/input-output-hk/cardano-js-sdk/blob/fbd851a3be28652356777d303f2025e9a56d7159/treefmt.toml#L8-L9).

I'm not so prepared to answer correctly to this question, but as far as I remember we use `eslint` with `prettier` rules, so I'm not surprised if formatting the code through `prettier` or `eslint` gives two different results.

A possible solution could be to configure `treefmt` to format only nix files and keep TypeScript formatting as is (in a new distinct activity anyway).

**Comment by michalrus** [comment]
@iccicci thank you :bow: 

@bernokl so considering ↑, I think we should exclude the whitespace changes in TypeScript files in this PR, as they would be reverted soon with `eslint`

**Review by gytis-ivaskevicius — APPROVED** [review]

**Comment by bernokl** [comment]
Merging as only remaining changes are in .nix and readme files

**Pull request merged**